src/frontends/tensorflow_common/src/op/matrix_band_part.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "openvino/op/logical_and.hpp"
+#include "openvino/op/less_equal.hpp"
+#include "openvino/op/unsqueeze.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/reshape.hpp"
+#include "utils.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_matrix_band_part_op(const NodeContext& node) {
+    default_op_checks(node, 3, {"MatrixBandPart"});
+    // Input tensor and parameters
+    auto input = node.get_input(0);
+
```

Comment:
please get `num_lower` and `num_upper` as follows, since they are inputs according to the specification https://www.tensorflow.org/api_docs/python/tf/raw_ops/MatrixBandPart:

```
    auto num_lower = node.get_input(1);
    auto num_upper = node.get_input(1);
```

No need to create `Constant` nodes for them below